actors/builtin/multisig/multisig_actor.go

```diff
@@ -167,24 +165,25 @@ func (a *MultiSigActor) RemoveSigner(rt vmr.Runtime, params *RemoveSigner) *vmr.
 	// Can only be called by the multisig wallet itself.
 	rt.ValidateImmediateCallerIs(rt.CurrReceiver())
 
-	h, st := a.State(rt)
-
-	if !st.isSigner(params.Signer) {
-		rt.AbortStateMsg("Party not found")
-	}
+	var st MultiSigActorState
+	rt.State().Transaction(&st, func() interface{} {
+		if !st.isSigner(params.Signer) {
+			rt.AbortStateMsg("Party not found")
+		}
 
-	newSigners := make([]addr.Address, 0, len(st.Signers))
-	for _, s := range st.Signers {
-		if s != params.Signer {
-			newSigners = append(newSigners, s)
+		newAuthorizedParties := make([]addr.Address, 0, len(st.Signers))
+		for _, s := range st.Signers {
+			if s != params.Signer {
+				newAuthorizedParties = append(newAuthorizedParties, s)
+			}
 		}
-	}
-	if params.Decrease || int64(len(st.Signers)-1) < st.NumApprovalsThreshold {
-		st.NumApprovalsThreshold = st.NumApprovalsThreshold - 1
-	}
-	st.Signers = newSigners
+		if params.Decrease || int64(len(st.Signers)-1) < st.NumApprovalsThreshold {
+			st.NumApprovalsThreshold = st.NumApprovalsThreshold - 1
+		}
+		st.Signers = newAuthorizedParties
```

Comment:
nit: should probably be `newSigners`, both here and the lines above :arrow_up:. I think this was missed while rebasing after https://github.com/filecoin-project/specs-actors/pull/35